proposals/stack-switching/Explainer.md

```diff
@@ -158,21 +158,25 @@ We add two new continuation heap types and their subtyping hierachy:
 
 ### Instructions
 
-The new instructions and their validation rules are as follows.
+The new instructions and their validation rules are as follows. To simplify the presentation, we write `C.types[$ct] = cont [t1*] -> [t2*]` where we really mean this:
+
+```
+C.types[$ct] = cont $ft
+C.types[$ft] = [t1*] -> [t2*]
+```
+
+This abbreviation will be formalized with an auxiliary function in the spec.
```

Comment:
FWIW, this may be possible without if we rely on a suitable normalisation invariant.